### ISSUE TITLE
Show One-Page Menu for Restaurant 

### ISSUE THREAD

**Issue by lolagarden** [SE701ProjectGroup4/git-brunching#181]
**User Story:**

As a user, I want to see a restaurant's menu, so I can decide where I want to make a booking.

**Acceptance Criteria:**

* When the user clicks the 'menu' button on a restaurant's card, a menu image is displayed in the popup
* The displayed menu is a _single_ dummy data menu image stored in the database 

**Additional Context:**
This ticket is blocked by: #183 & #184 
The functionality for if the menu is multiple pages is continued in another ticket (#182)

**Comment by jhsi517** [comment]
LGTM :)
